### DISCUSSION TITLE
Renovate Logs are grouping to a single line

### DISCUSSION THREAD

**Discussion by shenke-adsk** [mend/renovate-ce-ee#432] [Q&A] [answered]
We are using the Renovate CE image `ghcr.io/mend/renovate-ce:6.8.0-full`. We set this environment variable: `RENOVATE_LOG_FILE=/var/log/renovate/renovate_runtime.log` and we forward the logs to Splunk, hoping to take advantage the advanced query capabilities provided by Splunk. However, we are seeing that the logs are grouped on a single line. See the attached screenshot. It happens most of the time. 

![renovate logs](https://github.com/mend/renovate-ce-ee/assets/80284937/e777421a-dbeb-41f8-98b0-0a4c71a79be0)

**Comment by rarkins** [comment]
Isn't this a very standard "one line of JSON per line" log output? Ie the type which is perfect for splunk, and which splunk is parsing correctly in your screenshot?

**Reply by shenke-adsk** [reply]
Yes, the log entry 2 is. I was referring to entry 1. It is like this in most cases.

**Reply by rarkins** [reply, marked as answer]
Renovate outputs such logs one JSON object per line. Each line should be treated as a separate log by Splunk. Maybe the way you are forwarding them has a bug or problem

**Reply by shenke-adsk** [reply]
Yes, looks like it is issue in the forwarder.
